Makefile

```diff
@@ -32,3 +32,14 @@ test:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race ./...
 test-verbose:
 	CGO_LDFLAGS="-L`llvm-config --libdir`" go test -timeout 60s -race -v ./...
+
+docker-images: docks/build/Dockerfile docks/base/Dockerfile
```

Comment:
a.) Concerning the prerequisites, I do not think that we need that. "docker build" has its own cache mechanism.
b.) How about renaming "docker-images" to "docker-build-images"?